phpcs.xml.dist

```diff
@@ -3,4 +3,16 @@
     <config name="installed_paths" value="../../cakephp/cakephp-codesniffer" />
 
     <rule ref="CakePHP" />
+    <rule ref="SlevomatCodingStandard.Commenting.DocCommentSpacing">
```

Comment:
This rule would actually be adding to our coding standard.

Reply:
Does this new rule have a fixer?

Reply:
Yes, all changes in this PR are auto fixed, no manual changes required.